package.json

```diff
@@ -30,7 +30,6 @@
     "typecheck": "tsc",
     "test": "jest",
     "test:core": "jest packages/core",
-    "dev": "node ./scripts/dev.js",
```

Comment:
cc. @Andarist

Reply:
There wasn't actually a `scripts/dev.js` file so I was thinking the file was removed at some point but the script was forgotten

Reply:
Ah okay!

Reply:
ye, I used this to develop v4